pkg/hostagent/hostagent.go

```diff
@@ -424,6 +424,41 @@ func (a *HostAgent) Info(_ context.Context) (*hostagentapi.Info, error) {
 	return info, nil
 }
 
+func (a *HostAgent) installPackage() error {
```

Comment:
I would not install the package here. If we depend on rsync in the guest, we can install the package using cloud-init.

In our user-data (part of cidata.iso), we can add:

```
packages:
- rsync
```

This will install rsync using cloud-init when the instance is provisioned. For distros that do not support this, the provision part of the yaml can install rsync manually.